playbooks/bgp/prepare-bgp-computes.yaml

```diff
@@ -0,0 +1,71 @@
+---
+- name: Configure computes
+  hosts: "computes{{ networkers_bool | default(false) | bool | ternary(',networkers', '') }}"
+  tasks:
+    - name: Check default route corresponds with BGP
+      ansible.builtin.command:
+        cmd: "ip route show default"
+      register: initial_default_ip_route_result
```

Comment:
nit: private vars are better for temporary vars: `_initial_default_ip_route_result`

Reply:
done